preview_generator/file_converter.py

```diff
@@ -17,10 +16,3 @@ def txt_to_txt(text: typing.IO[typing.Any]) -> typing.IO[typing.Any]:
 def get_image_size(img: typing.IO[bytes]) -> typing.Tuple[int, int]:
     with WImage(file=img) as image:
         return image.size
-
-
-def get_pdf_size(pdf: typing.Union[str, typing.IO[bytes]], page_id: int) -> typing.Tuple[int, int]:
```

Comment:
Not needed anymore? Are we sure our users don't use this function?

Reply:
It's difficult to be sure, but the file_converter file is not used at all in any other part of the code since many years. And this method does use previous deprecated pdf lib to get the information.

For consistency, i did remove all code about file_converter for now.